api/src/main/resources/metadata/concepts_and_drugs/tools-2024/Concept.xml

```diff
@@ -735,4 +735,70 @@
         <concept  concept_id="199663"  retired="0"  datatype_id="3"  class_id="11"  is_set="0"  creator="1"  date_created="2025-07-01 10:38:28"  changed_by="1"  date_changed="2025-07-01 10:38:27"  uuid="5fffecdc-d0ad-40a9-b862-dd618079677f"/>
         <concept  concept_id="199664"  retired="0"  datatype_id="2"  class_id="11"  is_set="0"  creator="1"  date_created="2025-07-01 10:41:32"  changed_by="1"  date_changed="2025-07-01 10:41:32"  uuid="c3a62ca3-3810-4833-82b6-17f164b1ee09"/>
         <concept  concept_id="199665"  retired="0"  datatype_id="3"  class_id="11"  is_set="0"  creator="1"  date_created="2025-07-01 10:42:37"  changed_by="1"  date_changed="2025-07-01 10:42:37"  uuid="583537a4-9876-4560-a66e-1a62ec672658"/>
+
+        <concept  concept_id="199666"  retired="0"    datatype_id="4"  class_id="3"  is_set="0"  creator="1"  date_created="2025-11-11 23:21:11"     changed_by="1"  date_changed="2025-11-11 23:21:11"    uuid="937debd9-10f5-44e9-8985-5fc272687e0b"/>
```

Comment:
Did you consider changes from this PR https://github.com/METS-Programme/openmrs-module-ugandaemr/pull/838

Reply:
@slubwama going to update